packages/web/components/Technology/Details/Tables/Responsibles.js

```diff
@@ -36,7 +37,16 @@ const Responsibles = ({ data }) => {
 									<td>{item.name}</td>
 									<td>{item.email}</td>
 									<td>{item.phone_number}</td>
-									<td>{item.lattes_id}</td>
+									<td>
+										<Link
+											href={` http://lattes.cnpq.br/${item.lattes_id}`}
```

Comment:
It is best to return the full URL using a **computed attribute** on the server side

```js
// packages/api/app/Models/User.js

static get computed() {
	return ['full_name', 'lattes_url'];
}

getLattesUrl({ lattes_id }) {
	return `${cnpqBasePath}/${lattes_id}`;
}
```
Then...
```suggestion
											href={item.lattes_url}
```

Reply:
That's really necessary? I couldn't understand the need of doing this way, can you please explain?

Reply:
👍🏻